code/modules/research/xenoarchaeology/tools/tools_pickaxe_vr.dm

```diff
@@ -0,0 +1,126 @@
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Excavation pickaxes - sorted in order of delicacy. Players will have to choose the right one for each part of excavation.
+
+/obj/item/weapon/pickaxe/brush
```

Comment:
Yep. I had to do this due to not actually being able to change the tools_pickaxe.dm file or the icons/obj/xenoarchaeology.dmi file. 
This overwrites that and makes it so it uses the icons/obj/xenoarchaeology_vr.dmi file.